Services/Mail/classes/Mime/Transport/class.ilMailMimeTransportBase.php

```diff
@@ -148,8 +142,8 @@ final public function send(ilMimeMail $mail) : bool
 
         $this->mailer->Debugoutput = static function (string $message, $level) : void {
             if (
-                strpos($message, 'Invalid address') !== false ||
-                strpos($message, 'Message body empty') !== false
+                str_contains($message, 'Invalid address') ||
```

Comment:
`str_contains` is not supported in PHP 7.4 (https://www.php.net/manual/en/function.str-contains.php), which we'll have to support as well in ILIAS 8.